Assets/Scripts/ClientState.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+[Serializable]
+public class ClientState
```

Comment:
I realize you are probably just moving code around here, but anyway some inline documentation would be nice here!